nifi-nar-bundles/nifi-smb-bundle/nifi-smb-client-api/src/main/java/org/apache/nifi/services/smb/SmbListableEntity.java

```diff
@@ -154,7 +154,7 @@ public Record toRecord() {
         final Map<String, Object> record = new TreeMap<>();
         record.put("filename", getName());
         record.put("shortName", getShortName());
-        record.put("path", path);
+        record.put("path", getPath());
         record.put("identifier", getPathWithName());
         record.put("timestamp", getTimestamp());
         record.put("creationTime", getCreationTime());
```

Comment:
Please use the exactly same names for record fields and FF attributes:

- filename
- shortName
- path
- serviceLocation
- timestamp
- creationTime
- lastAccessTime
- changeTime
- size
- allocationSize

`identifier` should be omitted. `absolute.path` may also be not necessary because it can be calculated from `path` and `filename`.